ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -332,4 +330,13 @@ public String getDatabaseName() {
   public boolean isMaterializedView() {
     return table.isMaterializedView();
   }
+
+  public TableType tableType() {
```

Comment:
It might be useful as it tells if the corresponding view have data files associated or not. Anyways, Beacon guys looked at the doc and had no comments. Also, this change is not huge.